src/main/java/Duke.java

```diff
@@ -1,10 +1,155 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static Task[] tasks = new Task[100];
+
     public static void main(String[] args) {
+        Scanner in = new Scanner(System.in);
+        welcomeMessage();
+        boolean isOn = true;
+        int numOfCompletedTasks = 0;
+        int numOfTasks = 0;
+        int n = 0;
+        while(isOn && in.hasNextLine()) {
+            String line = in.nextLine();
+            String[] words = line.split(" ");
+            if(line.equals("bye")){
+                System.out.println("    --------------------------------------------------------------------------------");
+                System.out.println("    Bye. Hope to see you again soon!");
+                System.out.println(" ");
+                System.out.println("    --------------------------------------------------------------------------------");
+                isOn = false;
+            }
+            else if(words[0].equals("todo")){
+                String description = "";
+                int wordsCount = words.length;
+                for(String word:words){
+                    wordsCount--;
+                    if(word.equals("todo")){
+                        continue;
+                    }
+                    else if(wordsCount==0){
+                        description = description + word;
+                    }
+                    else{
+                        description = description + word + " ";
+                    }
+                }
+                tasks[n] = new Todo(description);
+                System.out.println("    --------------------------------------------------------------------------------");
+                System.out.println("    Got it. I've added this task:");
+                System.out.println("      "+tasks[n]);
+                n++;
+                System.out.println("    Now you have " + n +" tasks in the list.");
+                System.out.println(" ");
+                System.out.println("    --------------------------------------------------------------------------------");
+            }
+            else if(words[0].equals("deadline")){
+                String description = "";
+                String by = "";
+                boolean byFlag = false;
+                int wordsCount = words.length;
+                for(String word:words){
+                    wordsCount--;
+                    if(word.equals("/by")){
+                        byFlag = true;
+                        continue;
+                    }
+                    if(word.equals("deadline")){
+                        continue;
+                    }
+                    if(byFlag && wordsCount==0){
+                        by = by + word;
+                    }
+                    else if(byFlag){
+                        by = by + word + " ";
+                    }
+                    else{
+                        description = description + word + " ";
+                    }
```

Comment:
For the if-else statement, it would be better to follow the code standard requirement. 
eg. Between If and left bracket to need a space and else if should inline with the "}".